src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs

```diff
@@ -89,7 +89,7 @@ public IEnumerable<string> GetRenderContent(IGraphNode<LibraryDescription> root)
 
                 if (!_hideDependent)
                 {
-                    var dependents = string.Join(", ", dependenciesMap[description].Select(dep => dep.ToString()).OrderBy(name => name));
+                    var dependents = string.Join(", ", dependenciesMap[description].Select(dep => dep.Identity.ToString()).OrderBy(name => name));
```

Comment:
Not here. If you track down the source of the LibraryDescriptions' source, their Identities are already checked before the tree is created. *If* the identity is null, the failure will happen in the very beginning of dnu list.